grammars/javascript.cson

```diff
@@ -977,6 +1005,132 @@
         'name': 'variable.parameter.function.js'
       }
     ]
+  'arguments':
+    'patterns': [
+      {
+        'include': '#method_call'
+      }
+      {
+        'include': '#function_call'
+      }
+      {
+        'include': '#numbers'
+      }
+      {
+        'include': '#strings'
+      }
+      {
+        'include': '#object_variable'
+      }
+      {
+        'match': '\\[|\\]'
+        'name': 'meta.brace.square.js'
+      }
+      {
+        'match': ','
+        'name': 'meta.delimiter.object.comma.js'
+      }
+      {
+        'match': '\\{|\\}'
+        'name': 'meta.brace.curly.js'
+      }
+      {
+        'include': '#operators'
+      }
+      {
+        'include': '#comments'
+      }
+      {
+        'include': '#properties'
+      }
+      {
+        'match': '(?x) \\b[a-zA-Z_$]\\w* (?= \\s*, | \\) )'
```

Comment:
I don't really see the benefit of using `(?x)` here since the regex is pretty simple.  Same goes for below.